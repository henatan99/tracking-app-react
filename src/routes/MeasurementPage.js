import React from 'react';
import MeasurementForm from '../containers/measurementForm';

const MeasurementPage = () => (
  <MeasurementForm />
);

export default MeasurementPage;
