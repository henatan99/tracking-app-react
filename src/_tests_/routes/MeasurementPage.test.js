import React from 'react';
import { Provider } from 'react-redux';
import renderer from 'react-test-renderer';
import { MemoryRouter } from 'react-router-dom';
import MeasurementPage from '../../routes/MeasurementPage';
import createTestStore from '../factories/storeFactory';

describe('Measureds', () => {
    let store;
    beforeEach(() => {
      store = createTestStore();
    });
    test('should match with snapshot', () => {
      const tree = renderer
        .create(
          <Provider store={store}>
            <MemoryRouter>
              <MeasurementPage />
            </MemoryRouter>
          </Provider>,
        ).toJSON();
      expect(tree).toMatchSnapshot();
    });
  });
