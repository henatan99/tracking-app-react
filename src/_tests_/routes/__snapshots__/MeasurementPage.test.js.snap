// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Measureds should match with snapshot 1`] = `
<h3>
  No measurements
</h3>
`;
